bundles/org.openhab.ui/web/src/components/addons/addons-section.vue

```diff
@@ -75,6 +75,7 @@
     align-content flex-start
     align-items flex-end
     flex-wrap wrap
+    // gap 0.5rem
     padding-left calc(var(--f7-safe-area-left) + var(--f7-list-item-padding-horizontal))
```

Comment:
```suggestion
    padding-left calc(var(--f7-safe-area-left)
```
This was actually there before, but I don't know why there was this left padding... It looks better without.